### PR TITLE
Random refactors in EntityViewTile

### PR DESCRIPTION
**Pull request by phillipm**

### CODE REVIEW

File: app/src/org/commcare/views/EntityViewTile.java [hunk 1]
```diff
@@ -11,8 +8,8 @@
 import android.graphics.Point;
 import android.os.Build;
 import android.support.v4.util.Pair;
+import android.support.v4.widget.Space;
 import android.support.v7.widget.GridLayout;
-import android.support.v7.widget.Space;
```
Comment (phillipm) [inline app/src/org/commcare/views/EntityViewTile.java]: Is deprecated, replacing with `v4` version

File: app/src/org/commcare/views/EntityViewTile.java [hunk 2]
```diff
@@ -366,20 +356,20 @@ private View getView(Context context, GridStyle style, String fieldForm, String
...
                         Bitmap b = MediaUtil.inflateDisplayImage(getContext(), rowData,
                                 maxWidth, maxHeight, true);
-                        ((ImageView) retVal).setImageBitmap(b);
+                        ((ImageView)retVal).setImageBitmap(b);
                     }
                 }
                 break;
             case EntityView.FORM_AUDIO:
-                if (rowData != null & rowData.length() > 0) {
+                if (rowData != null && rowData.length() > 0) {
```
Comment (phillipm) [inline app/src/org/commcare/views/EntityViewTile.java]: Can only assume that `&` was meant to be `&&`

File: app/src/org/commcare/views/EntityViewTile.java [hunk 3]
```diff
@@ -392,19 +382,19 @@ private View getView(Context context, GridStyle style, String fieldForm, String
                 //so we've gotta mirror that for the search text. Booooo. I dunno if there's any
                 //other other side effects (newlines? nbsp?)
 
-                String htmlIfiedSearchField = searchField == null ? searchField : MarkupUtil.getSpannable(searchField).toString();
+                String htmlIfiedSearchField = searchField == null ? null : MarkupUtil.getSpannable(searchField).toString();
```
Comment (phillipm) [inline app/src/org/commcare/views/EntityViewTile.java]: `searchField` is always `null` in the first case of the ternary

**Comment by amstone326** [comment]
Looks like this has a test failure

**Comment by phillipm** [comment]
@amstone326 Yeah, looks like there was an issue with the script that checks out proper branches when running tests on Jenkins. Should be good to go now.

**Pull request merged**